### PR TITLE
screens 19-21 updated and added

### PR DESCRIPTION
**Pull request by mo3ni**
#79  
screen 19 updated 
screens 20-21 added

### CODE REVIEW

File: lib/models/task.dart [hunk 1]
```diff
@@ -1,10 +1,14 @@
 import 'dart:convert';
 
-import 'package:cloud_firestore/cloud_firestore.dart';
+//import 'package:cloud_firestore/cloud_firestore.dart';
```
Comment (atila29) [inline lib/models/task.dart]: why these comments?

File: lib/models/task.dart [hunk 2]
```diff
@@ -1,10 +1,14 @@
 import 'dart:convert';
```
Comment (atila29) [inline lib/models/task.dart:1]: Why change this file?

File: lib/screens/detail_page.dart [hunk 3]
```diff
@@ -24,6 +26,18 @@ class _DetailPageState extends State<DetailPage> {
   setSelectedRadio(int val) {
     setState(() {
       selectedRadio = val;
+      switch (selectedRadio) {
+        case 1:
```
Comment (atila29) [inline lib/screens/detail_page.dart]: Why do something in this case?

File: lib/screens/pay_screen.dart [hunk 4]
```diff
@@ -0,0 +1,216 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:job_bazaar/screens/applicant_screen.dart';
+import 'package:job_bazaar/screens/tasks_page.dart';
+import 'package:slider_button/slider_button.dart';
+
+class PayScreen extends StatefulWidget {
+  final DocumentSnapshot task;
```
Comment (atila29) [inline lib/screens/pay_screen.dart]: Use the Task model instead of DocumentSnapshot

**Review by atila29 — CHANGES_REQUESTED** [review]
Please update branch. And use Task model instead of DocumentSnapshot.

File: lib/models/task.dart (hunk 2 above)
Comment (elvarlax) [reply lib/models/task.dart:1]: Fix this

File: lib/models/task.dart [hunk 5]
```diff
@@ -114,33 +112,33 @@ class Task {
     if (identical(this, o)) return true;
 
     return o is Task &&
-      o.id == id &&
```
Comment (elvarlax) [inline lib/models/task.dart:117]: Don't change this, only the code you are adding

**Review by atila29 — APPROVED** [review]

**Review by elvarlax — APPROVED** [review]

**Pull request merged**